src/osd/OSDMap.h

```diff
@@ -893,7 +893,7 @@ class OSDMap {
   }
 
   bool exists(int osd) const {
-    //assert(osd >= 0);
+    //ceph_assert(osd >= 0);
```

Comment:
This one is clearly outdated because `osd >= 0` is checked in the body of the method.